### PR TITLE
Add some error logs and give conf templates for production cluster

### PR DESCRIPTION
**Pull request by dangleptr**
1. Configurations on production cluster are different with test cluster. So we give the templates for production cluster. Our users should use these templates as their default configurations when doing load test. 
2. Add more error logs on storaged

### CODE REVIEW

**Comment by dangleptr** [comment]
Jenkins go

**Comment by yixinglu** [comment]
Do these **production** configure files need to be copied to `/usr/local/nebula/etc` when we make nebula rpm/deb packages?

File: src/kvstore/Part.cpp [hunk 1]
```diff
@@ -441,6 +423,23 @@ bool Part::preProcessLog(LogID logId,
     return true;
 }
 
+ResultCode Part::toResultCode(raftex::AppendLogResult res) {
```
Comment (Shylock-Hg) [inline src/kvstore/Part.cpp]: Could make static.

File: etc/nebula-graphd.conf.production [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# Whether to run as a daemon process
+--daemonize=true
+# The file to host the process id
+--pid_file=pids/nebula-graphd.pid
+
+########## logging ##########
+# The directory to host logging files, which must already exists
+--log_dir=logs
+# Log level, 0, 1, 2, 3 for INFO, WARNING, ERROR, FATAL respectively
+--minloglevel=0
```
Comment (Shylock-Hg) [inline etc/nebula-graphd.conf.production]: Maybe not enable INFO level.

File: src/kvstore/Part.cpp (hunk 1 above)
Comment (dangleptr) [reply src/kvstore/Part.cpp]: It can't be static, because we need idStr_.

File: etc/nebula-graphd.conf.production (hunk 2 above)
Comment (dangleptr) [reply etc/nebula-graphd.conf.production]: On production cluster,  the default log level is INFO.

**Comment by dangleptr** [comment]
> Do these **production** configure files need to be copied to `/usr/local/nebula/etc` when we make nebula rpm/deb packages?

Good point.  WDYT?  @yixinglu

File: src/kvstore/Part.cpp [hunk 3]
```diff
@@ -441,6 +423,23 @@ bool Part::preProcessLog(LogID logId,
...
+    switch (res) {
+        case raftex::AppendLogResult::SUCCEEDED:
+            return ResultCode::SUCCEEDED;
+        case raftex::AppendLogResult::E_NOT_A_LEADER:
+            return ResultCode::ERR_LEADER_CHANGED;
+        case raftex::AppendLogResult::E_WRITE_BLOCKING:
+            return ResultCode::ERR_WRITE_BLOCK_ERROR;
+        default:
```
Comment (critical27) [inline src/kvstore/Part.cpp]: Maybe we can assign AppendLogResult::E_BUFFER_OVERFLOW to another ResultCode?

Comment (dangleptr) [reply src/kvstore/Part.cpp]: There are already too many codes inside ResultCode and cpp2::ErrorCode. 
We could check the internal consensus error code with this log.

**Comment by dangleptr** [comment]
> > Do these **production** configure files need to be copied to `/usr/local/nebula/etc` when we make nebula rpm/deb packages?
> 
> Good point. WDYT? @yixinglu

Let's open another pr if needed in the future.

**Pull request merged**